main.lua

```diff
@@ -9,7 +9,7 @@ function love.mousepressed(x, y, button, istouch)
     if menuActive then
         gooi.pressed()
     else
-        if aiType == nil or aiType == "random" and colorTurn == "white" then
+        if aiType == nil or aiType == "random" or aiType == "forward" and colorTurn == "white" then
```

Comment:
There might be something wrong here. I'm not sure exactly what this part is for, but I would think that `and` precedes `or`, meaning that we might need

```
if aiType == nil or aiType == "random" and colorTurn == "white" or aiType == "forward" and colorTurn == "white" then
```

or

```
if aiType == nil or ((aiType == "random" or aiType == "forward") and colorTurn == "white") then
```

That is, if I understood this part correctly.